test/hotspot/jtreg/vmTestbase/nsk/share/aod/AODRunnerArgParser.java

```diff
@@ -106,7 +106,12 @@ public String getTargetApp() {
             throw new TestBug("Target application isn't specified");
         }
 
-        return options.getProperty(TARGET_APP_PARAM);
+        String target = options.getProperty(TARGET_APP_PARAM);
```

Comment:
I think it is better to remove -classPath from parameters and 
use
 ${test.class.path}
directly in the test.

Reply:
The test is run by `nsk.share.aod.AODTestRunner` which runs test app specified by "target" argument.
Target apps have no system properties like test.class.path

Reply:
Thanks for explanation.